index.css

```diff
@@ -0,0 +1,187 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin-left: 10%;
+  margin-right: 10%;
+  border: 2px solid black;
+  margin-top: 20px;
+}
+
+.Container-fluid {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 12px;
+  border: 4px dashed brown;
+  height: 40px;
+  margin-bottom: 80px;
+}
+
+.top-nav {
+  background: #6e7072;
+  border: 2px solid black;
+}
+
+.top-part {
+  margin-left: 40%;
+  margin-bottom: 6px;
+}
+
+.bottom-part {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 20px;
+  padding-bottom: 20px;
+}
+
+.bottom-part p {
+  color: #fff;
+  font-size: 30px;
+  cursor: pointer;
+}
+
+.http-input {
+  border: 2px solid rgb(31, 28, 28);
+  border-radius: 8px;
+}
+
+.curved-input {
+  width: 60px;
+  height: 20px;
+  border-radius: 40%;
+}
+
+.logo {
+  margin-left: 8px;
+}
+
+.list-Container {
+  display: flex;
+  list-style: none;
+}
+
+.nav-items {
+  margin-right: 20px;
+}
+
+#List {
+  border-right: 4px solid;
+  border-color: rgb(219, 208, 208);
+  border-width: 4px;
+  padding-right: 10px;
+  cursor: pointer;
+}
+
+#Add {
+  border-right: 4px solid;
+  border-color: rgb(184, 97, 97);
+  border-width: 4px;
+  padding-right: 10px;
+  cursor: pointer;
+}
+
+#Link {
+  cursor: pointer;
+  color: rgba(30, 30, 255, 0.897);
+}
+
+.List-Items {
+  display: none;
+}
+
+.owesome-header {
+  margin-top: 20px;
+  margin-bottom: 30px;
+}
+
+.hold-collection {
+  width: 80%;
+  border: 2px solid black;
+}
+
+.add-books {
+  margin-left: 60px;
+  display: none;
+}
+
+.book-list {
+  margin-top: 12px;
+}
+
+.remove {
+  margin-left: 550px;
+}
+
+.newbooks {
+  display: flex;
+  flex-direction: column;
+}
+
+#title {
+  width: 30%;
+  margin-bottom: 12px;
+  margin-top: 12px;
+  border: 2px solid black;
+}
+
+#author {
+  width: 30%;
+  margin-top: 12px;
+  border: 2px solid black;
+}
+
+#btn {
+  width: 5%;
+  margin-left: 220px;
+  margin-top: -30px;
+  border-right: 4px solid black;
+  border-bottom: 4px solid black;
+}
+
+.collection tr:nth-child(2n + 0) {
+  background: red;
+}
+
+.collection tr:nth-child(2n + 1) {
+  background: #e4e5e6;
+}
+
+.Contact {
+  display: none;
+}
+
+.active {
+  display: block;
+}
+
+.con-head {
+  margin-bottom: 40px;
+}
+
+.con-ul {
+  list-style: none;
+}
+
+.or {
+  margin: 20px 0 20px 80px;
+  font-size: 40px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.copyright {
```

Comment:
- [ ] You have done an excellent job by implementing the requirements provided by the previous reviewer. However, your code is not completely finished. If you refer to the original wireframe, there are parts that still need to be addressed in order to adhere to the initial requirements. Please make sure to keep references to the original wireframe while completing this task. Kindy follow the instuction given by previous reviewer.


Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-12 at 15 15 59" src="https://user-images.githubusercontent.com/69011963/212043593-23f88791-438e-42f1-9bda-ee7127ce4098.png"> | <img  alt="Screenshot 2023-01-12 at 15 15 59" src="https://user-images.githubusercontent.com/69011963/212043571-747b748e-5e73-4856-ab3a-68f342dfbb80.png" width="1440"/>